### ISSUE TITLE
No sentries or any files added what so ever but not asking for steam guard code

### ISSUE THREAD

**Issue by rishav394** [Marc3842h/Titan#88]
## Description

Titan not asking for Steam Guard code

## Reproduction Steps

1. Downloaded the latest release 1.6
2. Ran Titan.exe
3. Added my account in account list
4. Added second account under commend tab
5. Hit commend

## Errors

```bash
22:37:49 [Main] DEBUG GC - artecnoob1 (Protected) - Successfully connected to St
eam. Logging in...
22:37:49 [Main] DEBUG GC - artecnoob1 (Protected) - No Sentry file found. Titan
will ask for a confirmation code...
22:37:49 [Main] DEBUG GC - artecnoob1 (Protected) - Logging in with Auth Code: n
ull / 2FA Code null / Hash: null / LoginID: (Removed LoginKey to protect privacy of user) / LoginKey: null
```

## System Information

**Operating System**: Windows 7
**Titan version**: Latest release 1.6

**Comment by rishav394** [comment]
```
16:21:14 [Main] DEBUG GC - artecnoob1 (Protected) - No Sentry file found. Titan
will ask for a confirmation code...
```
Nothing is happening hereafter. Tried the debug version(1.6.9-) also from Discord.

**Comment by Marc3842h** [comment]
Hey, thanks for checking out the version on the Discord. I'm currently in the process of fixing this and will release a RC on the Discord as soon as it's ready. If you want to help me, start Titan with `--steamkitdebug` and send me the output of the SteamKit debug listener.

Please note that you just leaked your account passwords and your API key by uploading your log files into this public issue section. I suggest you change them immediatly and start Titan using `--secure` to provide safe logfiles - alternatively you can also just send them to me over PM on Discord.

**Comment by rishav394** [comment]
Thanks for removing the log files and for the support. I'll make sure to use `--secure` next time and also use `--steamkitdebug` with Titan. Will send logs ASAP over PM on Discord.

**Comment by rishav394** [comment]
Logs from Titan 1.6.1 with `--secure` and `--steamkitdebug`

[AccountManager-20180315.log](https://github.com/Marc3842h/Titan/files/1814774/AccountManager-20180315.log)
[CommendTab-20180315.log](https://github.com/Marc3842h/Titan/files/1814775/CommendTab-20180315.log)
[GC - artecnoob1-20180315.log](https://github.com/Marc3842h/Titan/files/1814776/GC.-.artecnoob1-20180315.log)
[Quartz.Net-Scheduler-20180315.log](https://github.com/Marc3842h/Titan/files/1814777/Quartz.Net-Scheduler-20180315.log)
[SteamKit Debug Listener-20180315.log](https://github.com/Marc3842h/Titan/files/1814778/SteamKit.Debug.Listener-20180315.log)
[SWAHandle-20180315.log](https://github.com/Marc3842h/Titan/files/1814779/SWAHandle-20180315.log)
[ThreadManager-20180315.log](https://github.com/Marc3842h/Titan/files/1814780/ThreadManager-20180315.log)
[Titan Handler-20180315.log](https://github.com/Marc3842h/Titan/files/1814781/Titan.Handler-20180315.log)
[Titan-20180315.log](https://github.com/Marc3842h/Titan/files/1814782/Titan-20180315.log)
[Titan-20180315_001.log](https://github.com/Marc3842h/Titan/files/1814783/Titan-20180315_001.log)
[VictimTracker-20180315.log](https://github.com/Marc3842h/Titan/files/1814784/VictimTracker-20180315.log)

**Comment by Tylsew** [comment]
when this gonna be updated?

**Comment by rishav394** [comment]
Soon. He says he is working on it. Patience. Use virtual box Linux.

**Comment by Triple-P** [comment]
You can fix the account problem by updating all the nuget packages to the latest, at least for me it works ;)

**Comment by Marc3842h** [comment]
@Triple-P I don't think that may be the cause but I've updated the NuGet packages with commit 35648bf, just to be sure. People that have this issue including @rishav394 and @1n9i9c7om, would love if you could try out if that was really the cause.

**Comment by 1n9i9c7om** [comment]
I've tried compiling it and while I ran into issues while compiling, I finally got it.

First of all, the results: **It works!** The console output seems messed up, though: 
![Log](https://i.1n9i9c7om.com/JjUibk.png)

However, the issues I had while compiling:
* I couldn't compile it using the build.ps1 script. It always crashed due to this:
`Titan\packages\Eto.Forms.2.4.1\build\Eto.Forms.targets(5,14): error MSB4066: Das Update-At
tribut im <Compile>-Element ist unbekannt. [F:\Users\Nico\Desktop\Titan2\Titan\Titan\Titan.csproj]`
I've updated Visual Studio because I thought it might have been related to an older version of MSBuild, but that didn't fix the issue.
* However, the script downloaded all needed NuGet packages, so I could open the project in Visual Studio, where it actually did compile successfully.
* I had some issues with the protobuf library which I fixed by downgrading it back to 2.1.0 - but when I reverted back to the latest version (because I wanted to screenshot it), it just... worked. 

Off-Topic: I actually installed Titan on a Rock64 running Xenial Mate so I could connect on it using VNC and report people, I guess that's not needed anymore then. 😋
